arangod/Cluster/ActionBase.cpp

```diff
@@ -63,6 +63,22 @@ ActionBase::~ActionBase() {
   }
 }
 
+
+bool ActionBase::first() {
```

Comment:
Disagree with this.  There is no useful Action derivative that will not require first().  Revert to pure virtual.  _actionStarted is maintained by MaintenanceWorker::nextState() via the startStats() call.